src/index.js

```diff
@@ -2,7 +2,15 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
-import registerServiceWorker from './registerServiceWorker';
+// import registerServiceWorker from './registerServiceWorker';
```

Comment:
1) I have no idea what registerServiceWorker does, I think it was in the boilerplate, do you know?
2) If it's really useless then you can just delete the comments.

Reply:
Pretry sure service workers are related to pwas and loading content while offline. I left it in so it we choose to make this a pwa, it's already built in to the boilerplate